src/py42/sdk/queries/query_filter.py

```diff
@@ -1,8 +1,6 @@
 from collections import OrderedDict
```

Comment:
I believe we can also remove `OrderedDict` as all python dictionaries are ordered in python 3.6+